misc/python/materialize/benchmark_utils.py

```diff
@@ -0,0 +1,23 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+"""Benchmark utilities."""
+
+from materialize import buildkite
+
+
+def resolve_tag_of_common_ancestor() -> str:
+    if not buildkite.is_in_pull_request():
+        print("Not in a pull request, resolved common-ancestor to latest")
+        return "latest"
```

Comment:
This should return `None` , and then the caller should convert this to `latest` based on context. This will make this function more universal.

Reply:
I introduced `tag_when_on_default_branch` as parameter (but can change it to your solution if you prefer that).